src/plugins/telemetry/public/plugin.ts

```diff
@@ -38,6 +40,8 @@ export interface TelemetryServicePublicApis {
   getIsOptedIn: () => boolean | null;
   /** Is the user allowed to change the opt-in/out status? **/
   userCanChangeSettings: boolean;
+  /** Can phone-home telemetry calls be made? This depends on whether we have opted-in or if we are rendering a report */
+  canSendTelemetry: () => boolean;
```

Comment:
do we need to make this part of public contract or can it be kept internal ?

Reply:
Spoke offline and happy to defer this change to Telemetry team.